### ISSUE TITLE
clarify what platforms this is for

### ISSUE THREAD

**Issue by jmtd** [chocolate-doom/chocpkg#5]
Is the expectation that this will be used for windows builds, too? In which case I guess getting the cygwin environment up first is a prerequisite? Are we expecting to use this for the BSDs too? I.e. all platforms?

**Comment by fragglet** [comment]
Yes. It should be possible to get this working with Cygwin too although I haven't tried yet. I've done Windows (MingW) cross-compiles on Linux already. We should come up with per-platform setup instructions; it might be a good idea to put these as docs inside the repo and move them off the wiki.
